### PR TITLE
Content: Add FLYSPARKF4

### PR DESCRIPTION
**Pull request by VitroidFPV**

### CODE REVIEW

File: docs/wiki/boards/current/FLYSPARKF4/FLYSPARKF4.mdx [hunk 1]
```diff
@@ -0,0 +1,148 @@
+---
+sidebar_custom_props:
+  specs:
+    target: FLYSPARKF4
+    mcu: STM32F405RGT6
+    imu: ICM42688-P
+    osd: MAX7456EUI+ (SPI 2)
```
Comment (haslinghuis) [inline docs/wiki/boards/current/FLYSPARKF4/FLYSPARKF4.mdx:7]:
```suggestion
    osd: MAX7456EUI+ (SPI)
```

- suggest removing from darkmatter board too.
- suggest a smaller font size here

Comment (nerdCopter) [reply docs/wiki/boards/current/FLYSPARKF4/FLYSPARKF4.mdx:7]: the dynamic nature of this area always looks weird, no matter the width of the browser window :(
image

Comment (VitroidFPV) [reply docs/wiki/boards/current/FLYSPARKF4/FLYSPARKF4.mdx:7]: Yeah definitely needs more work on the info grid, I'll put this PR into draft in the meantime and see what I can do

Comment (nerdCopter) [reply docs/wiki/boards/current/FLYSPARKF4/FLYSPARKF4.mdx:7]: if the PR information is good, i don't mind approving/merging. can do the grid in another PR.

**Comment by nerdCopter** [comment]
re-open? this is still needed content not included in #480

**Comment by haslinghuis** [comment]
x

**Comment by VitroidFPV** [comment]
I pushed the changes to my branch already, I suppose I'll re-make this PR instead

**Comment by nerdCopter** [comment]
there is a way to repair
  https://gist.github.com/robertpainsi/2c42c15f1ce6dab03a0675348edd4e2c

**Comment by VitroidFPV** [comment]
It might be simpler to re-make the PR 😅

**Pull request closed without merging**